meinberlin/apps/budgeting/assets/BudgetingProposalListItem.jsx

```diff
@@ -9,7 +9,7 @@ const updatedOnStr = django.gettext('updated on')
 const createdOnStr = django.gettext('created on')
 
 export const BudgetingProposalListItem = (props) => {
-  const { proposal, permissions, tokenvoteApiUrl } = props
+  const { idx, proposal, permissions, queryParams, tokenvoteApiUrl } = props
```

Comment:
Ah! Good idea to use the index instead of searching for the idea in question in the api.
But doesn't this break with the other pagination? So, on page 2 of the list, it starts at the beginning again?

Reply:
ahh, I didn't take that into account, but I guess we can do `page * 15 + idx` ?